llvm/cppbuild.sh

```diff
@@ -125,4 +130,6 @@ case $PLATFORM in
         ;;
 esac
 
+cp -r $INSTALL_PATH/include/polly/isl $INSTALL_PATH/include/
+
```

Comment:
I had compilation problems with polly where their isl dependency headers were undiscoverable by the JavaCPP jnilib command. It looks to me like a regression from 11.0.0, but I will do more research. This is a temporary fix which works for now.

Reply:
@yukoba Here, it looks like Polly changed with LLVM 12. If you could help with that, please do! Nothing urgent though.

Reply:
@saudet
> Polly changed with LLVM 12

I see.

> Is there a way to add header search paths to the JavaCPP invocation without messing with CXX/CFLAGS?

I think you should answer this.
If I am not wrong, I think something related to ```@Platform```.

Reply:
If it's just for `LinkAllPasses.h`, just updating `polly/LinkAllPasses.h` with `polly/isl/LinkAllPasses.h` should work...

Reply:
I'm not sure its as simple as that... The file in question is the `$INSTALL_PATH/polly/isl/isl-noexceptions.h` which seems to be an entrypoint to libisl which doesn't use exceptions and this header is indirectly included through `polly/ScopInfo.h`. Rewriting `polly/ScopInfo.h` path to include polly would not work either because `polly/isl/isl-noexceptions.h` includes even more of isl's headers which would result in header search failure once that file is included.

Building without moving isl's headers yields the following compile error

```
In file included from /home/me/source/supergrecko/javacpp-presets/llvm/cppbuild/linux-x86_64/include/polly/ScopDetection.h:50,
                 from /home/me/source/supergrecko/javacpp-presets/llvm/cppbuild/linux-x86_64/include/polly/ScopInfo.h:20,
                 from /home/me/source/supergrecko/javacpp-presets/llvm/cppbuild/linux-x86_64/include/polly/ScopPass.h:20,
                 from /home/me/source/supergrecko/javacpp-presets/llvm/cppbuild/linux-x86_64/include/polly/Simplify.h:16,
                 from /home/me/source/supergrecko/javacpp-presets/llvm/cppbuild/linux-x86_64/include/polly/LinkAllPasses.h:20,
                 from /home/me/source/supergrecko/javacpp-presets/llvm/target/native/org/bytedeco/llvm/linux-x86_64/jniLLVM.cpp:134:
/home/me/source/supergrecko/javacpp-presets/llvm/cppbuild/linux-x86_64/include/polly/Support/ScopHelper.h:20:10: fatal error: isl/isl-noexceptions.h: No such file or directory
   20 | #include "isl/isl-noexceptions.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

I think we can solve this by adding it to the `@Platform` include path. Is it possible to use  https://github.com/bytedeco/javacpp/blob/master/src/main/java/org/bytedeco/javacpp/annotation/Platform.java#L62 relative from the install directory? We could alternatively rewrite every include directive in isl and polly's headers but that sounds very error prone.

Reply:
Ah, I see. For these cases, we can add more includePath in the pom.xml file like this:
https://github.com/bytedeco/javacpp-presets/blob/master/cpython/pom.xml#L36